remo/featuredrep/templates/featuredrep_alter.jinja

```diff
@@ -45,6 +45,13 @@
           </a>
           syntax.
         </p>
+      </div>
+      <div class="large-6 columns">
+        <input type="text" class="flat" value="{{ form.created_on.value() }}"
+                  name="created_on" />
```

Comment:
can you align the name attribute directly under the "type" attribute?
also, can you add a placeholder for the input field that mentions the format of the date that is expected? I think it's okay to not have a datepicker here, but I think we should show the expected format. This also means that we need to filter out the value if there is none set (otherwise it will overwrite the placeholder with "None").